src/request/TopLevelApi.js

```diff
@@ -156,6 +157,14 @@ class TopLevelApi extends RequestParser {
                 handler.run();
 
                 TopLevelApi.setLoading(false);
+
+                if (TopLevelApi.getDocumentWidth() > Constants.MIN_WIDTH_FOR_AUTOFOCUS) {
```

Comment:
You could extract this into it's own function `TopLevelApi.focus()` and then also call it from all other places where `passwordBox.focus()` is currently called.